docker/dev/README.md

```diff
@@ -0,0 +1,57 @@
+# Environment: Development
+
+This environment provides a mock backend for local development.  
+The mock backend simulates the LinID API metadata endpoints.
+
+## Quick Start
+
+```bash
+# Clean
+docker container prune -f
```

Comment:
If we want to build separetly the mock-backend, it will be nice te add the build command here. Even if the compose built it.
@jcabannes